scene/3d/navigation_region_3d.h

```diff
@@ -33,6 +33,7 @@
 
 #include "scene/3d/node_3d.h"
 #include "scene/resources/navigation_mesh.h"
+#include "servers/navigation_server_3d.h"
```

Comment:
It's not used in the header so can be moved to the .cpp.

Reply:
Whelp, seeing the compile error now I know again why it was there, because of the 3D gizmo.